int16 start_grasp
---
int16 success_grasp